app/src/main/java/com/example/fbufinalapp/CommonFunctions.java

```diff
@@ -0,0 +1,47 @@
+package com.example.fbufinalapp;
```

Comment:
Overall, great job centralizing some functions. That's a great coding habit that you should continue doing :)